apps/cookbook/src/app/examples/header-example/examples/action-group.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+
+import { BasePageExampleComponent } from '../../page-example/base-page-example.component';
+
+const config = {
+  template: `<kirby-page defaultBackHref="/">
+  <kirby-header title="Title" subtitle1="Subtitle one" subtitle2="Subtitle two">
+    <kirby-action-group *kirbyHeaderActions>
+      <button kirby-button (click)="actionClicked('Action 1')">
```

Comment:
Note to self: Dennis asked for this button to be att. lvl. 3, as that is the most plausible use case.